packages/cli/src/platform/$.ts

```diff
@@ -14,6 +14,7 @@ export class $ implements Command {
     const isHasEarlyAccessFeatureFlag = Boolean(argv.find((_) => _.match(/early-access-feature/)))
     if (!isHasEarlyAccessFeatureFlag) throw new EarlyAccessFlagError()
     const result = await dispatchToSubCommand(this.commands, argv)
+    // TODO: Consider removing JSON.stringify as it breaks if sub-command parse returns JSON.stringify
     return JSON.stringify(result)
```

Comment:
we could check if the type of result is `string` and otherwise stringify?

Reply:
That is a good solution!

Reply:
I will fix this in another PR as it touches files not relevant to this PR